# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/QuotaManager.h
# Commit: 868577854548
# Full Hash: 868577854548c7442c15ab9b1ad1213af187e01e
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2020-11-17 21:55:29
# Regressor Bug: 1671369
# File Overlap Count: 1
# Description:
#   Bug 1671369 - Change GetUsageForClient to return a UsageInfo. r=dom-workers-and-storage-reviewers,ttung
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D93777
# ==============================================================================

diff -r 5f347d3846aa -r 868577854548 dom/quota/QuotaManager.h
--- a/dom/quota/QuotaManager.h	Tue Nov 17 12:13:05 2020 +0000
+++ b/dom/quota/QuotaManager.h	Tue Nov 17 13:17:24 2020 +0000
@@ -219,9 +219,9 @@
                            const GroupAndOrigin& aGroupAndOrigin,
                            Client::Type aClientType);
 
-  bool GetUsageForClient(PersistenceType aPersistenceType,
-                         const GroupAndOrigin& aGroupAndOrigin,
-                         Client::Type aClientType, uint64_t& aUsage);
+  UsageInfo GetUsageForClient(PersistenceType aPersistenceType,
+                              const GroupAndOrigin& aGroupAndOrigin,
+                              Client::Type aClientType);
 
   void UpdateOriginAccessTime(PersistenceType aPersistenceType,
                               const GroupAndOrigin& aGroupAndOrigin);